conf/messages.en

```diff
@@ -544,6 +544,19 @@ guarantorName.error.length = Business name must be 182 characters or less
 guarantorName.error.invalidCharacter = Business name must not contain < and > and : and ;
 guarantorName.change.hidden = business name
 
+guarantorVat.3.title = What is the goods owner''s VAT registration number?
+guarantorVat.3.heading = What is the goods owner''s VAT registration number?
+guarantorVat.4.title = What is the transporter''s VAT registration number?
+guarantorVat.4.heading = What is the transporter''s VAT registration number?
```

Comment:
Can we use back ticks `’` instead?

Reply:
Updated on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/84/commits/f8220f4c3c2f8f72aa291200232000d80e7ffd49